aiida/cmdline/commands/cmd_archive.py

```diff
@@ -435,8 +437,9 @@ def _import_archive_and_migrate(archive: str, web_based: bool, import_kwargs: di
     from aiida.tools.archive.imports import import_archive as _import_archive
 
     archive_format = get_format()
+    filepath = ctx.obj['config'].get_option('storage.sandbox') or None
```

Comment:
Why do we have/need two different ways of loading this? I see below for the daemon workers you use:

```python
from aiida.manage.configuration import get_config_option

filepath_sandbox = get_config_option('storage.sandbox') or None
```

Reply:
I originally added this function (in 65c60d5ef8f3e1169a696fe095322e8163543877) as a shortcut to places in the code that are early in the bootstrapping process and so don't have the `Config` instance loaded yet. @chrisjsewell then refactored it in a8b3da36a0c3d7007519bb563774d38f2ce6b2e2 and moved it to a method on the `Manager`. Think it would make sense to discuss whether there is use to keeping both or we remove one of them. But that would be for another PR.